lib/websocket/receiver.js

```diff
@@ -94,9 +94,13 @@ class ByteParser extends Writable {
           const closeFrame = new WebsocketFrameSend(body)
 
           this.ws[kResponse].socket.write(
-            closeFrame.createFrame(opcodes.CLOSE)
+            closeFrame.createFrame(opcodes.CLOSE),
+            (err) => {
+              if (!err) {
+                this.ws[kSentClose] = true
```

Comment:
👍 but commit message is misleading:

1. It does not set `wasClean` but `Symbol('kSentClose')`.
2. When the callback is called it does not mean that the server received the frame.